src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -350,12 +352,9 @@ public LogContainerTestCallback(boolean buildSingleStringOutput, boolean buildFr
     @Override
     public void onNext(Frame frame) {
       String logStr = new String(frame.getPayload(), Charset.forName("UTF-8"));
-
-      // TODO: PF-423. Calling sysout.println here breaks the model of printing user-facing output
-      // from the command classes only.
-      // Revisit this once we have better model for centralizing output across all commands/rest of
-      // the codebase.
-      System.out.print(logStr); // write to stdout
+      PrintWriter err = Printer.getErr();
```

Comment:
I just did a little research into the differences between PrintWriter and PrintStream. It's not obvious why you went with the former here.

Reply:
I chose PrintWriter because I expect the CLI output to be primarily character data. I would have chosen PrintStream if I was planning to write out binary data.